cmd/worker/main.go

```diff
@@ -92,10 +93,20 @@ func saveResults(ctx context.Context, pkg *pkgecosystem.Pkg, dest resultBucketPa
 		}
 	}
 	if dest.fileWrites != "" {
-		err := resultstore.New(dest.fileWrites, resultstore.ConstructPath()).Save(ctx, pkg, dynamicResults.FileWrites)
+		err := resultstore.New(dest.fileWrites, resultstore.ConstructPath()).Save(ctx, pkg, dynamicResults.FileWritesSummary)
 		if err != nil {
 			return fmt.Errorf("failed to upload file write analysis to blobstore = %w", err)
 		}
+		for _, writeBufferPath := range dynamicResults.FileWriteBufferPaths {
+			writeBuffer, err := utils.ReadAndRemoveTempFile(writeBufferPath)
+			if err != nil {
+				log.Error("Could not read file", err)
```

Comment:
Should we be using fmt.Errorf for worker instead of log.Error? Where do the fmt.Error statements get logged to?

Reply:
`fmt.Errorf` doesn't actually log anything, it just creates an error to return. So you can use it when you want to return an error (and presumably stop the function) rather than just logging a message and continuing